app.js

```diff
@@ -35,6 +35,10 @@ app.use(helmet());
 app.use(cors());
 app.use(xss());
 
+//app.get('/', (req, res) => {
+// res.send('<h1>Jobs API</h1><a href="/api-docs">Documentation</a>');
+//});
```

Comment:
Avoid committing commented-out code; prefer to just delete it. Although, I don't see any handlers for the root route to replace this one. Am I missing something?